contracts/src/utils/math/alloy.rs

```diff
@@ -1,6 +1,17 @@
 //! Standard math utilities missing in `alloy_primitives`.
 use alloy_primitives::{uint, U256};
 
+/// Rounding modes for rounding operations.
+pub enum Rounding {
+    /// Toward negative infinity
+    Floor,
+    /// Toward positive infinity
+    Ceil,
+    /// Toward zero
+    Trunc,
+    /// Away from zero
+    Expand,
+}
```

Comment:
In response to https://github.com/OpenZeppelin/rust-contracts-stylus/pull/465#issuecomment-2560155192:

> > Should I implement something like mulDiv for U256 MATH?
> 
> ```solidity
> function _convertToShares(uint256 assets, Math.Rounding rounding) internal view virtual returns (uint256) {
>         return assets.mulDiv(totalSupply() + 10 ** _decimalsOffset(), totalAssets() + 1, rounding);
>     }
> ```

Let's not do this as part of this PR. Remove changes to this file, and implement the necessary logic in the contract.